### DISCUSSION TITLE
Add Microsoft's 'Photos' application to the 'open with' submenu

### DISCUSSION THREAD

**Discussion by jczanfona** [aleksey-hoffman/sigma-file-manager#135] [Q&A]
I couldn't figure out how to do it ...
[App link](https://www.microsoft.com/en-us/p/microsoft-photos/9wzdncrfjbh4#activetab=pivot:overviewtab)
Thanks in advance.

**Comment by aleksey-hoffman** [comment]
It's difficult to open installed Microsoft Store apps since Windows doesn't have proper APIs for this.
I will try to implement a way to select installed apps some time later.

**Comment by jczanfona** [comment]
ok, thanks in advance!
